examples/run_mda.py

```diff
@@ -11,5 +11,8 @@
 )
 
 mmc = CMMCorePlus()
-mmc.loadSystemConfiguration("demo")
-mmc.run_mda(sequence)
+mmc.loadSystemConfiguration("demo_config.cfg")
```

Comment:
this now only works if you cd into the `examples` directory first (I tend to run `python examples/whatever.py`)... can you make it `str(Path(__file__).parent / "demo_config.cfg")`